site2/docs/reference-configuration.md

```diff
@@ -125,6 +125,7 @@ BookKeeper is a replicated log storage system that Pulsar uses for durable stora
 |dbStorage_rocksDB_numLevels||-1|
 |dbStorage_rocksDB_numFilesInLevel0||4|
 |dbStorage_rocksDB_maxSizeInLevel1MB||256|
+|skipListSizeLimit|Heap memory limitation of `EntryMemTable`, The `EntryMemTable` holds in-memory representation to the entries not-yet flushed. When asked to flush, current EntrySkipList is moved to snapshot and is cleared.|64MB|
```

Comment:
This configuration is for `sortedLedgerStroage`, it won't work for `dbLedgerStorage`. 
However, Pulsar uses dbLedgerStorage by default, and we are not recommended users to use `sortedLedgerStorage`

Reply:
There are `skipListArenaChunkSize` and `skipListArenaMaxAllocSize` in the reference-config, and it's all for `EntryMemTable`, I think whether it's recommended or not, add the configuration into  reference-config is required, in case of someone to ask it.

Reply:
OK, I'll close the PR